src/codebook/settings/functionality/updateFilters.js

```diff
@@ -0,0 +1,30 @@
+import { select as d3select } from "d3";
+
+export default function updateFilters(codebook) {
+  const filterCheckBoxes = codebook.settings.wrap.selectAll(
+    ".column-table .Filter input"
+  );
+
+  //Add click functionality to each list item.
+  filterCheckBoxes.on("change", function() {
+    const filters = filterCheckBoxes
+      .filter(function() {
+        return d3.select(this).property("checked");
+      })
+      .data()
+      .map(d => d.Column);
+    codebook.config.filters = filters.map(d => {
```

Comment:
I think this is probably fine for now actually. We're going to re-work filters in a separate issue.